controllers/logging/elasticsearch_controller.go

```diff
@@ -52,6 +61,9 @@ func (r *ElasticsearchReconciler) Reconcile(request ctrl.Request) (ctrl.Result,
 		return ctrl.Result{}, nil
 	}
 
+	// keep track of the fact that we processed this elasticsearch for future events and for mapping
+	registerCRNamespacedName(&registeredElasticsearches, request)
```

Comment:
this shouldn't be necessary -- we'll only need to mark the ES nodes to be restarted based on the secret name...
the secret name and namespace should match the elasticsearch CR that contains the node information. we should be able to work from there.